chatkit-core/src/main/kotlin/com/pusher/chatkit/messages/api/MessagePartBodyApiType.kt

```diff
@@ -0,0 +1,8 @@
+package com.pusher.chatkit.messages.api
+
+internal data class MessagePartBodyApiType(
```

Comment:
I think we can drop the `Body` part from the naming in this package.

Reply:
Have dropped the word `body` from the file name, not the package name - i hope that's what you intended?

Reply:
I still see `Body` in the file name. I meant both the class and the file name. I think package name was fine.